mypy/test/testfinegrained.py

```diff
@@ -149,6 +149,7 @@ def get_options(self, source: str, testcase: DataDrivenTestCase, build_cache: bo
         options.use_fine_grained_cache = self.use_cache and not build_cache
         options.cache_fine_grained = self.use_cache
         options.local_partial_types = True
+        options.export_types = True
```

Comment:
What's the purpose of this change?

Reply:
Starting the daemon with this option set to `True` is cleaner for `inspect` unit tests, I made this explicit now (so it will not slow down other fine grained tests).